Resources/Locale/en-US/interaction/verbs/noop.ftl

```diff
@@ -16,6 +16,12 @@ interaction-Pet-success-self-popup = You pet {THE($target)} on {POSS-ADJ($target
 interaction-Pet-success-target-popup = {THE($user)} pets you on {POSS-ADJ($target)} head.
 interaction-Pet-success-others-popup = {THE($user)} pets {THE($target)}.
 
+interaction-PetAnimal-name = {interaction-Pet-name}
```

Comment:
Is this localization strings in localized strings?

Reply:
yes, fluent supports referencing other locale strings from other locale strings

Reply:
Very cool